test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
@byxorna I poked around this a bit more and I think our usage of dynamic frequency scaling is causing this behavior. The reported CPU speed is "changing" in the lshw XML files depending on the load of the system. 

Additionally, it looks like the current default behavior is to capture CPU information from the first socket and persist that information for all sockets.

As a result of these two behaviors, the existing `LshwHelperSpec` tests will always fail.

TBH, I'm not sure what is "expected behavior" for this. Should collins assume the max clock speed for `CpuSpeedGhz` for all sockets? Should collins honor the data in the lshw XML files and permit different clock speeds per socket?

I'm leaning towards opening a separate GH issue for this because it is unrelated to any of the GPU changes.

Thoughts?